style.css

```diff
@@ -0,0 +1,70 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-family: 'Crete Round', serif;
+}
+
+html,
+body {
+  height: 100%;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  padding: 20px 40px;
+  background-color: #3c3a39;
+  color: #fff;
+}
+
+header a {
+  font-family: 'Roboto', sans-serif;
+  text-decoration: none;
+  font-size: 18px;
+  font-weight: 700;
+  color: #fff;
+}
+
+.hero {
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+  background-color: #1c1a19;
+  padding: 150px 24px;
+  background-image: url(./images/header.svg);
+  background-repeat: no-repeat;
+  background-position: bottom right;
+  height: 100%;
+}
+
+.home-header {
+  color: #fff;
+}
+
+.home-header h1 {
+  color: #ff6b00;
+  font-size: 56px;
+  font-weight: 400;
+}
+
+.home-header h3 {
+  font-family: 'Roboto', sans-serif;
+  font-size: 24px;
+  font-weight: 700;
+}
+
+.home-header p {
+  font-size: 16px;
+}
+
+.socials {
+  display: flex;
+  list-style: none;
+  gap: 10px;
+  margin-left: 3px;
+}
+
+.socials li i {
+  color: #fff;
+}
```

Comment:
Please i'd suggest you use the background color in the Figma design for the socials icons which is `#DBD8D7`, and also add spacing between the social icons similar to what we have in the design :